base/README.md

```diff
@@ -10,10 +10,15 @@ This bundle provides the core LaTeX kernel. In addition to this bundle,
 a minimal LaTeX system also needs the files contained in the
 
 - LaTeX team documentation (`doc`)
+- L3 programming layer (`l3kernel`)
+- L3 backend interfaces (`l3backend`)
+- L3 unicode support (`unicode-data`)
```

Comment:
Well it's 'core Unicode support' (used by plain XeTeX/LuaTeX), but I'll let you off ;)